content/how-to/dedicated-hosting/manage-local-users-in-the-myrackspace-portal/index.md

```diff
@@ -1,44 +1,52 @@
 ---
 permalink: manage-local-users-in-the-myrackspace-portal
 audit_date: '2019-12-12'
-title: Manage local users in the MyRackspace portal
+title: Manage Local Users in the MyRackspace Portal
 type: article
 created_date: '2019-12-12'
 created_by: Chad Sterling
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2022-11-11'
```

Comment:
We might as well go ahead and update the date to today if other changes are going to be made or if it will be merged.

Reply:
Date is changed